popSimManu.tex

```diff
@@ -941,13 +946,60 @@ \section*{Acknowledgments}
 \section*{Supplemental Figures}
 \begin{figure}
 \begin{center}
-\includegraphics[width=0.8\linewidth]{display_items/homo_sapiens_mask_Gutenkunst.png}
+\includegraphics[width=0.8\linewidth]{display_items/HomSap_AE9K19_chr22_map_AI.pdf}
+\caption{\textbf{Validating the \texttt{SLiM} engine backend under a genetic map}.
+Here we validate our integration of the \texttt{SLiM} \citep{haller2019tree,haller2019slim} engine backend. 
+We show quantile-quantile plots between \texttt{SLiM} and \texttt{msprime} engines for three 
+population genetic summary statistics: \textit{r}\textsuperscript{2}, Tajimas's pi, and Tajima's D. Additionally, we 
+show runtimes for generating each simulation replicate. Data were generated by simulating 100 replicates 
+of human chromosome 22 under the AncientEurasia\_9K19 model \citep{kamm2019efficiently} and using the
+HapMapII\_GRCh37 genetic map \citep{international2007second}. From top to bottom we show results 
+using population-size rescaling for three scaling factors: Q=1, Q=10, and Q=100.
```

Comment:
Population-size scaling isn't menioned outside these two figures. Is this enough information to convey to the reader what is being plotted?

Reply:
No, I think we need to address this further, and I'm happy to take a stab at that.